static/proxy/template/layout/pkg/wiremock/client.go

```diff
@@ -78,6 +78,14 @@ func DoRequestWithStreamSize(request *http.Request) ([]byte, int, error) {
 
 func doRequest(request *http.Request) (*http.Response, error) {
 	if md, ok := metadata.FromIncomingContext(request.Context()); ok {
+		for mdKey, mdValue := range md {
+			if mdKey == ":authority" {
```

Comment:
SHOULD

Please move `":authority"` to const - at 82 and 89 lines.
Thank you for the PR.

Reply:
Refactored loop so as not to use :authority twice